external-crates/move/crates/move-disassembler/src/disassembler.rs

```diff
@@ -48,6 +50,11 @@ pub struct DisassemblerOptions {
     /// Print the locals inside each function body.
     #[clap(long = "print-locals")]
     pub print_locals: bool,
+
+    /// Maximum size of the output. If the output exceeds this size, the disassembler will return
+    /// an error.
+    #[clap(long = "max-output-size")]
+    pub max_output_size: Option<usize>,
```

Comment:
If this parameter is not specified, is the output size unbounded? I am assuming the unbounded was the only option before this PR and it could be useful to avoid specifying max output size in some cases (e.g., disassembling locally to a file).

Reply:
Correct -- as `clap` will use the `Option::default()` for this which will be `None`.